src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/Features/Ilr Changes/learner_changes_contract_type.feature

```diff
@@ -0,0 +1,75 @@
+﻿@learner_changes_contract_type
+Feature: Learner changes contract type
+
+Scenario: DPP-965_01 - Levy apprentice, provider edits contract type (ACT) in the ILR, previous on-programme and English/math payments are refunded and repaid according to latest contract type
+    Given The learner is programme only DAS
+	And levy balance > agreed price for all months
+	And the apprenticeship funding band maximum is 9000
+
+	And the following commitments exist:
+		| commitment Id | version Id | ULN       | start date | end date   | framework code | programme type | pathway code | agreed price | status | effective from | effective to |
+		| 1             | 1          | learner a | 01/08/2017 | 01/08/2018 | 403            | 2              | 1            | 9000         | Active | 01/08/2017     |              |
+        
+	When an ILR file is submitted for period R01 with the following data:
+        | ULN       | learner type           | agreed price | start date | planned end date | actual end date | completion status | aim type         | aim sequence number | aim rate | framework code | programme type | pathway code | contract type | contract type date from | contract type date to |
+        | learner a | programme only non-DAS | 9000         | 06/08/2017 | 20/08/2018       |                 | continuing        | programme        | 2                   |          | 403            | 2              | 1            | Non-DAS       | 06/08/2017              | 20/08/2018            |
+        | learner a | programme only non-DAS |              | 06/08/2017 | 20/08/2018       |                 | continuing        | maths or english | 1                   | 471      | 403            | 2              | 1            |               |                         |                       |
+        
+    And an ILR file is submitted for period R03 with the following data:
+        | ULN       | learner type       | agreed price | start date | planned end date | actual end date | completion status | aim type         | aim sequence number | aim rate | framework code | programme type | pathway code | contract type | contract type date from | contract type date to |
+        | learner a | programme only DAS | 9000         | 06/08/2017 | 20/08/2018       |                 | continuing        | programme        | 2                   |          | 403            | 2              | 1            | DAS           | 06/08/2017              | 20/08/2018            |
+        | learner a | programme only DAS |              | 06/08/2017 | 20/08/2018       |                 | continuing        | maths or english | 1                   | 471      | 403            | 2              | 1            |               |                         |                       |
+
+	Then the provider earnings and payments break down as follows:
+        | Type                                    | 08/17  | 09/17  | 10/17  | 11/17    |
```

Comment:
These payments disagree with those given in the acceptance criteria, but Tom has confirmed with Andy (the P.O.) that the payments in the two scenarios in this pull request are correct. Ideally the acceptance criteria in the story should have been updated.